Namespace shift

wboykinm opened the pull request.

nhusher commented on package.json:
```diff
@@ -1,7 +1,7 @@
   "name": "tilesplash",
   "description": "A light and quick nodejs webserver for serving mapbox vector tiles (and topojson) from a postgis backend",
-  "version": "2.1.2",
+  "version": "2.2.0",
```
I'd recommend bumping the major version number rather than the minor because the API has changed from connection strings to config objects.

wboykinm replied:
Oooh, yeah, more changes coming too. Def going for v3 before I publish.

wboykinm commented:
`pg`'s update for v7 is requiring some semi-serious rethinking of connection string usage. We're shifting to [something like this invocation](https://github.com/brianc/node-pg-pool#note) for our own apps going forward:

```
var url = require('url')
var params = url.parse(process.argv[2])
var auth = params.auth.split(':')

var config = {
  user: auth[0],
  password: auth[1],
  host: params.hostname,
  port: params.port,
  database: params.pathname.split('/')[1],
  ssl: true
}

var app = new Tilesplash(config)
```

wboykinm commented:
@nhusher requesting one final look-over for obvious nonsense.

kachkaev commented:
This is terrific! Just want to thank you guys for this update, because it's so in time for me! :fireworks: :tada: :balloon:
